src/tools/nbench/nbench.cpp

```diff
@@ -71,14 +71,23 @@ vector<PerfShape> to_perf_shape(shared_ptr<Function> f,
     return result;
 }
 
+string get_common_name(string name)
+{
+    if (name.compare(0, 4, "NNP_") == 0)
+    {
+        name[3] = '.';
+    }
+    return name.substr(0, name.find('_'));
+}
+
 multimap<size_t, string> aggregate_timing_details(const vector<PerfShape>& perf_data)
 {
     unordered_map<string, size_t> timing;
     unordered_map<string, size_t> count;
     for (const PerfShape& p : perf_data)
     {
         auto node = p.get_node();
-        string op = node->get_name().substr(0, node->get_name().find('_'));
+        string op = get_common_name(node->get_name());
```

Comment:
I think we can just use node->description() here instead of trimming the node's name. get_name() might return something without an underscore if the user has set a friendly name.

Reply:
much better. Thank you.